ansible/roles/prep.dirs/tasks/repo.yml

```diff
@@ -4,38 +4,40 @@
     insertafter: EOF
     line: "include('{{ mock_dir }}/glusterfs-nightly-repo.tpl')"
   with_items:
-    - "{{ mock_dir }}/centos-stream+epel-next-{{ item }}-x86_64.cfg"
-    - "{{ mock_dir }}/rhel-{{ item }}-x86_64.cfg"
+    - "{{ mock_dir }}/centos-stream+epel-next-{{ os_vers }}-x86_64.cfg"
```

Comment:
This would mean that we search for _centos-stream+epel-next-40-x86_64.cfg_ when invoked with `make rpms.fedora`(or _centos-stream+epel-next-39-x86_64.cfg_ when invoked with `make rpms.fedora vers=39`), right? Similarly for _rhel_ templates.

Reply:
Yeah, that's right. We use the value of os_vers, which means that we only generate configs for the os version we're building for. This value is set in the playbooks. Do we generate fedora and rhel config when building for fedora?

Reply:
Ok, I do see what you mean about how this would look for a file that doesn't exist when building for fedora. Honestly, the entirely of repo.yml isn't necessary for fedora though. Maybe we can skip this if we're not building for centos or rhel?

This also fails if ran from a playbook that does not set `os_vers`, like ansible/build.srpm.yml. I'd have to figure out a way around that as well.